packages/venia-concept/src/components/PurchaseHistoryPage/PurchaseHistory/purchaseHistory.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { shape, string, arrayOf } from 'prop-types';
+import { List } from '@magento/peregrine';
+
+import PurchaseHistoryItem from '../PurchaseHistoryItem';
+import classify from 'src/classify';
+import defaultClasses from './purchaseHistory.css';
+import Filter from '../Filter';
+import { PURCHASE_HISTORY_ITEM_PROP_TYPES } from './constants';
+
+class PurchaseHistory extends Component {
+    static propTypes = {
+        classes: shape({
+            body: string,
+            item: string,
+            filterContainer: string,
+            itemsContainer: string
+        }),
+        items: arrayOf(shape(PURCHASE_HISTORY_ITEM_PROP_TYPES))
```

Comment:
```suggestion
        items: arrayOf(shape({ id: number.isRequired }))
```

Let's not be overly prescriptive with the types here. For an object in the `items` array, the only prop this component cares about is `id`, which it uses for keys. Aside from that, this component doesn't know or care what props `PurchaseHistoryItem` expects.